src/com/chessclock/android/ChessClock.java

```diff
@@ -318,20 +338,11 @@ public void CheckForNewPrefs() {
 		
 		alertTone = prefs.getString("prefAlertSound", Settings.System.DEFAULT_RINGTONE_URI.toString());
 		
-		/** Check for a new delay style */
-		String new_delay = prefs.getString("prefDelay","None");
-		if (new_delay.equals("")) {
```

Comment:
Please correct me if I'm wrong, but I don't think delay can ever be an empty string, so this check seemed unnecessary.

Reply:
I think you're right.